katrain/i18n/locales/fr/LC_MESSAGES/katrain.po

```diff
@@ -594,3 +594,12 @@ msgstr ""
 "Le coup {move} a été annulé car il perdait {points_lost:.1f} points. Vous "
 "pouvez survoler le coup pour voir la réfutation. Veuillez jouer un nouveau "
 "coup."
+
```

Comment:
also please just add english and run i18n.py, this adds a TODO tags to other languages.